[
{"message":"file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n","level":"FATAL","type":"PLAYGROUND","test":"asyncify-boot","plugin":"dropbox-backup","details":"file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n","log":"/tmp/tmp.Bnle0wpnGw"}
,
{"message":"Unparenthesized `a ? b : c ? d : e` is not supported. Use either `(a ? b : c) ? d : e` or `a ? b : (c ? d : e)` in /wordpress/wp-content/plugins/dailymotion-search-and-publish-videos/classes/ocv_dm.php on line 47","level":"FATAL","type":"PHP","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 01:52:55 UTC] PHP Fatal error:  Unparenthesized `a ? b : c ? d : e` is not supported. Use either `(a ? b : c) ? d : e` or `a ? b : (c ? d : e)` in /wordpress/wp-content/plugins/dailymotion-search-and-publish-videos/classes/ocv_dm.php on line 47","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"Unparenthesized `a ? b : c ? d : e` is not supported. Use either `(a ? b : c) ? d : e` or `a ? b : (c ? d : e)` in /wordpress/wp-content/plugins/dailymotion-search-and-publish-videos/classes/ocv_dm.php on line 47","level":"FATAL","type":"PHP","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 01:53:02 UTC] PHP Fatal error:  Unparenthesized `a ? b : c ? d : e` is not supported. Use either `(a ? b : c) ? d : e` or `a ? b : (c ? d : e)` in /wordpress/wp-content/plugins/dailymotion-search-and-publish-videos/classes/ocv_dm.php on line 47","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"Unparenthesized `a ? b : c ? d : e` is not supported. Use either `(a ? b : c) ? d : e` or `a ? b : (c ? d : e)` in /wordpress/wp-content/plugins/dailymotion-search-and-publish-videos/classes/ocv_dm.php on line 47","level":"FATAL","type":"PHP","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 01:53:11 UTC] PHP Fatal error:  Unparenthesized `a ? b : c ? d : e` is not supported. Use either `(a ? b : c) ? d : e` or `a ? b : (c ? d : e)` in /wordpress/wp-content/plugins/dailymotion-search-and-publish-videos/classes/ocv_dm.php on line 47","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 01:55:24 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'submitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: submitted, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3142): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(398): deepi_select_posts_by_status('submitted')\n#4 /wordpress/wp-content/plugins/deepi/functions.php(412): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/back/settings.php(3): deepi_all_to_index()\n#6 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#8 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('deepi/deepi.php')\n#9 /internal/eval.php(20): activate_plugin('deepi/deepi.php')\n#10 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'submitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_to_index, deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 01:55:24 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'unsubmitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: unsubmitted, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3142): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(399): deepi_select_posts_by_status('unsubmitted')\n#4 /wordpress/wp-content/plugins/deepi/functions.php(412): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/back/settings.php(3): deepi_all_to_index()\n#6 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#8 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('deepi/deepi.php')\n#9 /internal/eval.php(20): activate_plugin('deepi/deepi.php')\n#10 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'unsubmitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_to_index, deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 01:55:24 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'crawled' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: crawled, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3142): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(400): deepi_select_posts_by_status('crawled')\n#4 /wordpress/wp-content/plugins/deepi/functions.php(412): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/back/settings.php(3): deepi_all_to_index()\n#6 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#8 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('deepi/deepi.php')\n#9 /internal/eval.php(20): activate_plugin('deepi/deepi.php')\n#10 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'crawled' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_to_index, deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 01:55:24 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'error' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: error, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3142): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(401): deepi_select_posts_by_status('error')\n#4 /wordpress/wp-content/plugins/deepi/functions.php(412): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/back/settings.php(3): deepi_all_to_index()\n#6 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#8 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('deepi/deepi.php')\n#9 /internal/eval.php(20): activate_plugin('deepi/deepi.php')\n#10 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'error' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_to_index, deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 01:55:24 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'submitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: submitted, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3142): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(398): deepi_select_posts_by_status('submitted')\n#4 /wordpress/wp-content/plugins/deepi/back/settings.php(4): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('deepi/deepi.php')\n#8 /internal/eval.php(20): activate_plugin('deepi/deepi.php')\n#9 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'submitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 01:55:24 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'unsubmitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: unsubmitted, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3142): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(399): deepi_select_posts_by_status('unsubmitted')\n#4 /wordpress/wp-content/plugins/deepi/back/settings.php(4): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('deepi/deepi.php')\n#8 /internal/eval.php(20): activate_plugin('deepi/deepi.php')\n#9 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'unsubmitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 01:55:24 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'crawled' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: crawled, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3142): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(400): deepi_select_posts_by_status('crawled')\n#4 /wordpress/wp-content/plugins/deepi/back/settings.php(4): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('deepi/deepi.php')\n#8 /internal/eval.php(20): activate_plugin('deepi/deepi.php')\n#9 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'crawled' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 01:55:24 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'error' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: error, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3142): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(401): deepi_select_posts_by_status('error')\n#4 /wordpress/wp-content/plugins/deepi/back/settings.php(4): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('deepi/deepi.php')\n#8 /internal/eval.php(20): activate_plugin('deepi/deepi.php')\n#9 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'error' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 01:55:25 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'submitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: submitted, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3142): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(398): deepi_select_posts_by_status('submitted')\n#4 /wordpress/wp-content/plugins/deepi/functions.php(412): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/back/settings.php(3): deepi_all_to_index()\n#6 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#7 /wordpress/wp-settings.php(526): include_once('/wordpress/wp-c...')\n#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#11 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'submitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_to_index, deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 01:55:25 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'unsubmitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: unsubmitted, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3142): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(399): deepi_select_posts_by_status('unsubmitted')\n#4 /wordpress/wp-content/plugins/deepi/functions.php(412): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/back/settings.php(3): deepi_all_to_index()\n#6 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#7 /wordpress/wp-settings.php(526): include_once('/wordpress/wp-c...')\n#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#11 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'unsubmitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_to_index, deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 01:55:25 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'crawled' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: crawled, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3142): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(400): deepi_select_posts_by_status('crawled')\n#4 /wordpress/wp-content/plugins/deepi/functions.php(412): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/back/settings.php(3): deepi_all_to_index()\n#6 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#7 /wordpress/wp-settings.php(526): include_once('/wordpress/wp-c...')\n#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#11 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'crawled' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_to_index, deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 01:55:25 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'error' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: error, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3142): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(401): deepi_select_posts_by_status('error')\n#4 /wordpress/wp-content/plugins/deepi/functions.php(412): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/back/settings.php(3): deepi_all_to_index()\n#6 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#7 /wordpress/wp-settings.php(526): include_once('/wordpress/wp-c...')\n#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#11 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'error' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_to_index, deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 01:55:25 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'submitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: submitted, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3142): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(398): deepi_select_posts_by_status('submitted')\n#4 /wordpress/wp-content/plugins/deepi/back/settings.php(4): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#6 /wordpress/wp-settings.php(526): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'submitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 01:55:25 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'unsubmitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: unsubmitted, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3142): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(399): deepi_select_posts_by_status('unsubmitted')\n#4 /wordpress/wp-content/plugins/deepi/back/settings.php(4): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#6 /wordpress/wp-settings.php(526): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'unsubmitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 01:55:25 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'crawled' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: crawled, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3142): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(400): deepi_select_posts_by_status('crawled')\n#4 /wordpress/wp-content/plugins/deepi/back/settings.php(4): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#6 /wordpress/wp-settings.php(526): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'crawled' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 01:55:25 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'error' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: error, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3142): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(401): deepi_select_posts_by_status('error')\n#4 /wordpress/wp-content/plugins/deepi/back/settings.php(4): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#6 /wordpress/wp-settings.php(526): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'error' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 01:55:34 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'submitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: submitted, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3142): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(398): deepi_select_posts_by_status('submitted')\n#4 /wordpress/wp-content/plugins/deepi/functions.php(412): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/back/settings.php(3): deepi_all_to_index()\n#6 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#8 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('deepi/deepi.php')\n#9 /internal/eval.php(20): activate_plugin('deepi/deepi.php')\n#10 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'submitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_to_index, deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 01:55:34 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'unsubmitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: unsubmitted, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3142): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(399): deepi_select_posts_by_status('unsubmitted')\n#4 /wordpress/wp-content/plugins/deepi/functions.php(412): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/back/settings.php(3): deepi_all_to_index()\n#6 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#8 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('deepi/deepi.php')\n#9 /internal/eval.php(20): activate_plugin('deepi/deepi.php')\n#10 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'unsubmitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_to_index, deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 01:55:34 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'crawled' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: crawled, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3142): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(400): deepi_select_posts_by_status('crawled')\n#4 /wordpress/wp-content/plugins/deepi/functions.php(412): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/back/settings.php(3): deepi_all_to_index()\n#6 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#8 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('deepi/deepi.php')\n#9 /internal/eval.php(20): activate_plugin('deepi/deepi.php')\n#10 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'crawled' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_to_index, deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 01:55:34 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'error' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: error, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3142): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(401): deepi_select_posts_by_status('error')\n#4 /wordpress/wp-content/plugins/deepi/functions.php(412): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/back/settings.php(3): deepi_all_to_index()\n#6 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#8 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('deepi/deepi.php')\n#9 /internal/eval.php(20): activate_plugin('deepi/deepi.php')\n#10 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'error' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_to_index, deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 01:55:34 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'submitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: submitted, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3142): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(398): deepi_select_posts_by_status('submitted')\n#4 /wordpress/wp-content/plugins/deepi/back/settings.php(4): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('deepi/deepi.php')\n#8 /internal/eval.php(20): activate_plugin('deepi/deepi.php')\n#9 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'submitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 01:55:34 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'unsubmitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: unsubmitted, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3142): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(399): deepi_select_posts_by_status('unsubmitted')\n#4 /wordpress/wp-content/plugins/deepi/back/settings.php(4): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('deepi/deepi.php')\n#8 /internal/eval.php(20): activate_plugin('deepi/deepi.php')\n#9 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'unsubmitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 01:55:34 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'crawled' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: crawled, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3142): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(400): deepi_select_posts_by_status('crawled')\n#4 /wordpress/wp-content/plugins/deepi/back/settings.php(4): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('deepi/deepi.php')\n#8 /internal/eval.php(20): activate_plugin('deepi/deepi.php')\n#9 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'crawled' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 01:55:34 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'error' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: error, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3142): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(401): deepi_select_posts_by_status('error')\n#4 /wordpress/wp-content/plugins/deepi/back/settings.php(4): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('deepi/deepi.php')\n#8 /internal/eval.php(20): activate_plugin('deepi/deepi.php')\n#9 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'error' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 01:55:35 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'submitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: submitted, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3142): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(398): deepi_select_posts_by_status('submitted')\n#4 /wordpress/wp-content/plugins/deepi/functions.php(412): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/back/settings.php(3): deepi_all_to_index()\n#6 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#7 /wordpress/wp-settings.php(526): include_once('/wordpress/wp-c...')\n#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#11 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'submitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_to_index, deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 01:55:35 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'unsubmitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: unsubmitted, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3142): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(399): deepi_select_posts_by_status('unsubmitted')\n#4 /wordpress/wp-content/plugins/deepi/functions.php(412): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/back/settings.php(3): deepi_all_to_index()\n#6 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#7 /wordpress/wp-settings.php(526): include_once('/wordpress/wp-c...')\n#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#11 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'unsubmitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_to_index, deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 01:55:35 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'crawled' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: crawled, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3142): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(400): deepi_select_posts_by_status('crawled')\n#4 /wordpress/wp-content/plugins/deepi/functions.php(412): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/back/settings.php(3): deepi_all_to_index()\n#6 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#7 /wordpress/wp-settings.php(526): include_once('/wordpress/wp-c...')\n#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#11 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'crawled' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_to_index, deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 01:55:35 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'error' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: error, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3142): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(401): deepi_select_posts_by_status('error')\n#4 /wordpress/wp-content/plugins/deepi/functions.php(412): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/back/settings.php(3): deepi_all_to_index()\n#6 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#7 /wordpress/wp-settings.php(526): include_once('/wordpress/wp-c...')\n#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#11 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'error' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_to_index, deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 01:55:35 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'submitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: submitted, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3142): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(398): deepi_select_posts_by_status('submitted')\n#4 /wordpress/wp-content/plugins/deepi/back/settings.php(4): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#6 /wordpress/wp-settings.php(526): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'submitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 01:55:35 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'unsubmitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: unsubmitted, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3142): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(399): deepi_select_posts_by_status('unsubmitted')\n#4 /wordpress/wp-content/plugins/deepi/back/settings.php(4): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#6 /wordpress/wp-settings.php(526): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'unsubmitted' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 01:55:35 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'crawled' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: crawled, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3142): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(400): deepi_select_posts_by_status('crawled')\n#4 /wordpress/wp-content/plugins/deepi/back/settings.php(4): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#6 /wordpress/wp-settings.php(526): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'crawled' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such column: deepi_status.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 01:55:35 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select `ID` from `wp_posts` where `deepi_status` = 'error' AND  `post_status` = 'publish' and `ping_status` = 'open' ;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select `ID` from `wp_posts` where `deepi_status` = :param0  AND  `post_status` = :param1  and `ping_status` = :param2  ; | parameters: error, publish, open</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: deepi_status.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3142): WP_SQLite_DB->query('select `ID` fro...')\n#2 /wordpress/wp-content/plugins/deepi/functions.php(197): wpdb->get_results('select `ID` fro...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/deepi/functions.php(401): deepi_select_posts_by_status('error')\n#4 /wordpress/wp-content/plugins/deepi/back/settings.php(4): deepi_all_status()\n#5 /wordpress/wp-content/plugins/deepi/deepi.php(32): include('/wordpress/wp-c...')\n#6 /wordpress/wp-settings.php(526): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query select `ID` from `wp_posts` where `deepi_status` = 'error' AND  `post_status` = 'publish' and `ping_status` = 'open' ; made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/deepi/deepi.php'), include('/plugins/deepi/back/settings.php'), deepi_all_status, deepi_select_posts_by_status, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"Array and string offset access syntax with curly braces is no longer supported in /wordpress/wp-content/plugins/dynamic-headers/custom-header.php on line 491","level":"FATAL","type":"PHP","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 01:56:11 UTC] PHP Fatal error:  Array and string offset access syntax with curly braces is no longer supported in /wordpress/wp-content/plugins/dynamic-headers/custom-header.php on line 491","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"Array and string offset access syntax with curly braces is no longer supported in /wordpress/wp-content/plugins/dynamic-headers/custom-header.php on line 491","level":"FATAL","type":"PHP","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 01:56:19 UTC] PHP Fatal error:  Array and string offset access syntax with curly braces is no longer supported in /wordpress/wp-content/plugins/dynamic-headers/custom-header.php on line 491","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"Array and string offset access syntax with curly braces is no longer supported in /wordpress/wp-content/plugins/dynamic-headers/custom-header.php on line 491","level":"FATAL","type":"PHP","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 01:56:28 UTC] PHP Fatal error:  Array and string offset access syntax with curly braces is no longer supported in /wordpress/wp-content/plugins/dynamic-headers/custom-header.php on line 491","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"Function register_sidebar_widget is <strong>deprecated</strong> since version 2.8.0! Use wp_register_sidebar_widget() instead. in /wordpress/wp-includes/functions.php on line 6114","level":"DEPRECATED","type":"PHP","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 02:01:41 UTC] PHP Deprecated:  Function register_sidebar_widget is <strong>deprecated</strong> since version 2.8.0! Use wp_register_sidebar_widget() instead. in /wordpress/wp-includes/functions.php on line 6114","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"Function register_widget_control is <strong>deprecated</strong> since version 2.8.0! Use wp_register_widget_control() instead. in /wordpress/wp-includes/functions.php on line 6114","level":"DEPRECATED","type":"PHP","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 02:01:41 UTC] PHP Deprecated:  Function register_widget_control is <strong>deprecated</strong> since version 2.8.0! Use wp_register_widget_control() instead. in /wordpress/wp-includes/functions.php on line 6114","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"Function register_sidebar_widget is <strong>deprecated</strong> since version 2.8.0! Use wp_register_sidebar_widget() instead. in /wordpress/wp-includes/functions.php on line 6114","level":"DEPRECATED","type":"PHP","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 02:01:53 UTC] PHP Deprecated:  Function register_sidebar_widget is <strong>deprecated</strong> since version 2.8.0! Use wp_register_sidebar_widget() instead. in /wordpress/wp-includes/functions.php on line 6114","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"Function register_widget_control is <strong>deprecated</strong> since version 2.8.0! Use wp_register_widget_control() instead. in /wordpress/wp-includes/functions.php on line 6114","level":"DEPRECATED","type":"PHP","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 02:01:53 UTC] PHP Deprecated:  Function register_widget_control is <strong>deprecated</strong> since version 2.8.0! Use wp_register_widget_control() instead. in /wordpress/wp-includes/functions.php on line 6114","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"Function register_sidebar_widget is <strong>deprecated</strong> since version 2.8.0! Use wp_register_sidebar_widget() instead. in /wordpress/wp-includes/functions.php on line 6114","level":"DEPRECATED","type":"PHP","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 02:02:05 UTC] PHP Deprecated:  Function register_sidebar_widget is <strong>deprecated</strong> since version 2.8.0! Use wp_register_sidebar_widget() instead. in /wordpress/wp-includes/functions.php on line 6114","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"Function register_widget_control is <strong>deprecated</strong> since version 2.8.0! Use wp_register_widget_control() instead. in /wordpress/wp-includes/functions.php on line 6114","level":"DEPRECATED","type":"PHP","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 02:02:05 UTC] PHP Deprecated:  Function register_widget_control is <strong>deprecated</strong> since version 2.8.0! Use wp_register_widget_control() instead. in /wordpress/wp-includes/functions.php on line 6114","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"Array and string offset access syntax with curly braces is no longer supported in /wordpress/wp-content/plugins/dropbox-backup/libs/wpadm.server.main.class.php on line 86","level":"FATAL","type":"PHP","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 02:02:41 UTC] PHP Fatal error:  Array and string offset access syntax with curly braces is no longer supported in /wordpress/wp-content/plugins/dropbox-backup/libs/wpadm.server.main.class.php on line 86","log":"/logs/plugins/d/dropbox-backup/error.json"}
,
{"message":"Array and string offset access syntax with curly braces is no longer supported in /wordpress/wp-content/plugins/dropbox-backup/libs/wpadm.server.main.class.php on line 86","level":"FATAL","type":"PHP","test":"asyncify-boot","plugin":"dropbox-backup","details":"[15-Apr-2025 02:02:49 UTC] PHP Fatal error:  Array and string offset access syntax with curly braces is no longer supported in /wordpress/wp-content/plugins/dropbox-backup/libs/wpadm.server.main.class.php on line 86\n\n    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n  [cause]: Error: Error when executing the blueprint step #1 ({\"step\":\"installPlugin\",\"pluginData\":{\"resource\":\"wordpress.org/plugins\",\"slug\":\"dropbox-backup\"},\"options\":{\"activate\":true}}) : PHP.run() failed with exit code 255 and the following output: \n      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)\n      ... 2 lines matching cause stack trace ...\n      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n    [cause]: PHPExecutionFailureError: PHP.run() failed with exit code 255 and the following output: \n        at PHP.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/universal/index.js:987:19)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async wr (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:33:39)\n        at async Object.gt [as installPlugin] (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:974:42)\n        at async g (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12517:30)\n        at async Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12446:23)\n        at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)\n        at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:227:48)\n        at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n        at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n      response: PHPResponse {\n        httpStatusCode: 500,\n        headers: {\n          'x-powered-by': [ 'PHP/8.0.30-dev' ],\n          'set-cookie': [ 'PHPSESSID=1746a187236696ae5beb01f4c769bb53; path=/' ],\n          expires: [ 'Thu, 19 Nov 1981 08:52:00 GMT' ],\n          'cache-control': [ 'no-store, no-cache, must-revalidate' ],\n          pragma: [ 'no-cache' ],\n          'content-type': [ 'text/html; charset=UTF-8' ]\n        },\n        bytes: Uint8Array(0) [],\n        exitCode: 255,\n        errors: ''\n      },\n      source: 'request'\n    }\n  }\n}\n\n","log":"/logs/plugins/d/dropbox-backup/error.json"}
]
